lib/slax/slack.ex

```diff
@@ -65,7 +65,7 @@ defmodule Slax.Slack do
   Sends a message to slack with the given url
   """
   def send_message(url, message) do
-    request = Jason.encode!(message)
+    request = "{\"text\": \"#{message}\"}"
```

Comment:
Not sure what `message` contains, but might be possible for it to escape by accident i think?

Reply:
Good point, not sure why this was changed, I'll look into it

Reply:
https://api.slack.com/methods/chat.postMessage

looks like it expects a json object here, so we'd want to revert this change or just change this to:

```
Jason.encode!(%{text: message})
```

https://api.slack.com/interactivity/slash-commands#enabling-interactivity-with-slash-commands__best-practices


This change might have been made due to the way Poker shouldn't send some messages to the channel? There's a `Message Visibility` that explains it a bit more in the link above.